server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -229,13 +238,57 @@ export function create(
 		},
 	);
 
+	/**
+	 * Delete a document
+	 */
+	router.delete(
+		"/:tenantId/document/:id",
+		validateRequestParams("tenantId", "id"),
+		validateTokenClaimsScopes(DocDeleteScopeType),
+		verifyStorageToken(tenantManager, config, tokenManager),
+		throttle(generalTenantThrottler, winston, tenantThrottleOptions),
+		async (request, response, next) => {
+			const documentId = getParam(request.params, "id");
+			const tenantId = getParam(request.params, "tenantId");
+			const lumberjackProperties = getLumberBaseProperties(documentId, tenantId);
+			Lumberjack.info(`Received document delete request.`, lumberjackProperties);
+
+			// TODO: Replace dummy implementation bellow.
```

Comment:
>// TODO: Replace dummy implementation bellow. [](http://example.com/codeflow?start=3&length=45)

(nit) the comment is more pertinent to `DocumentDeleteService` class. The implementation here is complete.